src/module/data/SR5ItemDataWrapper.ts

```diff
@@ -249,9 +249,13 @@ export class SR5ItemDataWrapper extends DataWrapper<ShadowrunItemData> {
         return this.getData()?.armor?.value ?? 0;
     }
 
+    getHardened(): boolean {
+        return this.getData()?.armor?.hardened || false;
```

Comment:
Code style suggestion: I recommend using the null coalesce operator (`??`) here, similar to line 249.